stable/ksoc-plugins/templates/ksoc-runtime/runtime-profile-config.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.ksocRuntime.detectReachableVulnerabilities }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ksoc-runtime-profile-configuration
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app_name: ksoc-sync
```

Comment:
Isn't it a part of `ksoc-runtime`?

Reply:
It depends on how we define and name this new functionality, but it requires the KSOC runtime plugin. In particular, eBPF probes that generate events used to build runtime profiles, which are eventually cross-referenced with package vulnerabilities.

Reply:
Shouldn't the `app_name` label be `ksoc-runtime` then?

Reply:
Good catch! Fixed, thanks!